src/components/NewGalaxyModal.js

```diff
@@ -61,19 +79,45 @@ export const NewGalaxyModal = _.flow(
       ['launchWarn', () => {
         return h(GalaxyLaunchButton, { app, onClick: onDismiss })
       }],
+      ['paused', () => {
+        return h(Fragment, [
+          h(ButtonPrimary, { style: { marginRight: 'auto' }, onClick: () => setViewMode('deleteWarn') }, ['Delete']),
+          h(ButtonSecondary, { style: { marginRight: '1rem' }, onClick: () => { resumeGalaxy() } }, ['Resume'])
+        ])
+      }],
       [Utils.DEFAULT, () => {
         return !!app ?
           h(Fragment, [
-            h(ButtonSecondary, { style: { marginRight: 'auto' }, onClick: () => setViewMode('deleteWarn') }, ['Delete']),
-            h(ButtonPrimary, { onClick: () => setViewMode('launchWarn') }, ['Launch Galaxy'])
+            (app.status === 'RUNNING') && h(Fragment, [h(ButtonSecondary, { style: { marginRight: 'auto' }, onClick: () => setViewMode('deleteWarn') }, ['Delete']),
+              h(ButtonSecondary, { style: { marginRight: '1rem' }, onClick: () => { pauseGalaxy() } }, ['Pause']),
+              h(ButtonPrimary, { onClick: () => setViewMode('launchWarn') }, ['Launch Galaxy'])]),
+            (app.status === 'STOPPED') && h(Fragment, [h(ButtonSecondary, { style: { marginRight: 'auto' }, onClick: () => setViewMode('deleteWarn') }, ['Delete']),
+              h(ButtonPrimary, { style: { marginRight: '1rem' }, onClick: () => { resumeGalaxy() } }, ['Resume'])]),
+            (app.status === 'ERROR') && h(ButtonSecondary, { style: { marginRight: 'auto' }, onClick: () => setViewMode('deleteWarn') }, ['Delete'])
           ]) :
```

Comment:
This should probably be a `Utils.switchCase` instead of this series of conditionals wrapped in a `Fragment`.
Also, be mindful of consistency with line breaks in your arrays, for instance there should be a linebreak after the open bracket for the children on 91 since they span multiple lines.

Reply:
Also, I notice that the Delete button always seems to be there, should it be rendered outside of any conditional so it's not repeated?

Reply:
I think there are some instances where the Delete Button shouldn't be there: Starting/Stopping states stand out to me.